opam

```diff
@@ -14,9 +14,10 @@ build-test: [
   [ "ocaml" "pkg/pkg.ml" "test" ]
 ]
 depends: [
-  "ctypes" { >= "0.11.0" }
+  "ctypes" { >= "0.12.0" }
   "ctypes-foreign" { >= "0.4.0" }
   "hex" { >= "1.0.0" }
+  "integers" { >= "0.2.2" }
```

Comment:
I tried compiling with `0.2.0` and it failed but I haven't looked further.  I'll do that now.